lib/ansible/cli/__init__.py

```diff
@@ -115,6 +115,15 @@ def initialize_locale():
     HAS_ARGCOMPLETE = False
 
 
+def _handle_deprecated_kwarg(func):
+    "Handle the deprecation of create_new_password in build_vault_ids."
+    def wrapper(*args, **kwargs):
```

Comment:
To avoid overriding `build_vault_ids`'s name/docstring with `wrapper`, decorate with functools.wraps
```suggestion
    @functools.wraps
    def wrapper(*args, **kwargs):
```